packages/react/src/Blankslate/Blankslate.tsx

```diff
@@ -143,16 +158,31 @@ function Blankslate({border, children, narrow, spacious}: BlankslateProps) {
 export type VisualProps = React.PropsWithChildren
 
 function Visual({children}: VisualProps) {
-  return <span className="Blankslate-Visual">{children}</span>
+  const enabled = useFeatureFlag('primer_react_css_modules')
+  return (
+    <span
+      className={cx('Blankslate-Visual', {
+        [classes.Visual]: enabled,
```

Comment:
_Note for feature flags only_: This is cool! It's a little bit cheating because the component already had classNames 😅